README.md

```diff
@@ -86,24 +90,31 @@ where:
 -   `110` is the node id of the bridge app assigned during the commissioning
 -   `1` is the endpoint of the configured device
 
-Level control:
+Level (brightness) control:
 ```bash
 chip-tool levelcontrol move-to-level 100 0 0 0 110 1
 ```
 
 Color control:
 ```bash
+# hue
 chip-tool colorcontrol move-to-hue 50 0 0 0 0 110 1
+# saturation
 chip-tool colorcontrol move-to-saturation 60 0 0 0 110 1
+# hue + saturation
 chip-tool colorcontrol move-to-hue-and-saturation 50 60 0 0 0 110 1
```

Comment:
The examples are not very clear in terms of whether they are using the Matter range or the Tapo range, as they are situated within both ranges. Could you please either ensure that the examples use identical values, such as `200` `254`, or add a sentence to explain that they are following the Matter range?

```suggestion
# hue
chip-tool colorcontrol move-to-hue 200 0 0 0 0 110 1
# saturation
chip-tool colorcontrol move-to-saturation 254 0 0 0 110 1
# hue + saturation
chip-tool colorcontrol move-to-hue-and-saturation 200 254 0 0 0 110 1
```